src/config/config_manager.h

```diff
@@ -185,45 +60,40 @@ class ConfigManager {
     virtual ~ConfigManager();
 
     /// \brief Returns the name of the config file that was used to launch the server.
```

Comment:
IMHO copying comment from an interface usually leads to inconsistency: base class already contains info what this method should do, therefore `override` is sufficient, especially in this case.